e2e/healthcheck.test.js

```diff
@@ -0,0 +1,17 @@
+import { test, expect } from "@playwright/test";
+import { getApiContext, URL } from "./common.js";
+
+let apiContext;
+
+test.beforeAll(async ({ playwright }) => {
```

Comment:
The bellow 3 sections need to be part of `test.describe` with a valid name of the test suite
test.beforeAll, test.afterAll, test

Reply:
I am referring to this verison if [test.describe](https://playwright.dev/docs/api/class-test#test-describe-1)